apps/api-gateway/src/connection/connection.controller.ts

```diff
@@ -77,39 +77,44 @@ export class ConnectionController {
     })
     @ApiQuery({
         name: 'pageNumber',
+        example: '1',
         type: Number,
         required: false
       })
       @ApiQuery({
         name: 'searchByText',
+        example: '',
         type: String,
         required: false
       })
       @ApiQuery({
         name: 'pageSize',
+        example: '10',
         type: Number,
         required: false
       })
       @ApiQuery({
         name: 'sorting',
+        example: 'createDateTime',
         type: String,
         required: false
       })
       @ApiQuery({
         name: 'sortByValue',
+        example: 'DESC',
         type: String,
         required: false
       })
-    @ApiResponse({ status: 200, description: 'Success', type: AuthTokenResponse })
+    @ApiResponse({ status: HttpStatus.OK, description: 'Success', type: AuthTokenResponse })
     async getConnections(
         @Query() getAllConnectionsDto: GetAllConnectionsDto,
         @User() user: IUserRequest,
         @Param('orgId') orgId: string,
         @Res() res: Response
-    ): Promise<Response> {
+    ): Promise<IResponseType> {
 
         const { pageSize, searchByText, pageNumber, sorting, sortByValue } = getAllConnectionsDto;
-        const connectionSearchCriteria: IConnectionSearchinterface = {
+        const connectionSearchCriteria: ConnectionSearchinterface = {
```

Comment:
Let's have consistency in naming conventions for Interfaces. We have decided to use "I" as a prefix and then interface name. No need to add word "interface".